tutorecommerce/templates/ecommerce/build/ecommerce/Dockerfile

```diff
@@ -5,7 +5,7 @@ FROM docker.io/ubuntu:20.04 as minimal
 RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/var/cache/apt,sharing=locked \
```

Comment:
@shadinaif I've found that the build gets stuck asking for user input which won't work during `docker build`, unless the `DEBIAN_FRONTEND=noninteractive` is set.

See https://techoverflow.net/2019/05/18/how-to-fix-configuring-tzdata-interactive-input-when-building-docker-images/

I've noticed that this variable is set as well in Tutor Discovery:

 - https://github.com/overhangio/tutor-discovery/blame/af757b3e4c31e0f41c9c8014b1094dd94e2decbc/tutordiscovery/templates/discovery/build/discovery/Dockerfile#L5

Reply:
Done